databend_py/client.py

```diff
@@ -271,6 +271,21 @@ def upload(self, file_name, table_name, file_type=None):
         file_data = self.get_file_data(file_name)
         self.sync_csv_file_into_table(file_name, file_data, table_name, file_type)
 
+    def upload_to_stage(self, file_name):
```

Comment:
if i want upload my local file to a stage, maybe i'd hope to specify the target location like a `cp` command.

```
upload_to_stage(self, local_path, target_path)
```

besides, I guess a file descriptor is better than a local path, so I can upload file from the arbitary file objects:

```
def upload_to_stage(self, f, stage_path):
```

Reply:
Make sense.

Reply:
I did a test and we need know the file content or we get problem when `copy into` 🤔

Reply:
> if we do not care much about the target path's file name, maybe we could just generate some randomized file names to avoid conflict. maybe looks like:
> 
> ```
> def upload_to_stage(self, f, target_path=None):
>   if target_path is None:
>     target_path = gen_random_path()
> ```

It's not a good way to modify the file name for user?

Reply:
ignore my previous comment, just taking the `target_path` from user side is better IMO.

Reply:
Maybe we can make filename as option:

```
 if file_name is None:
            file_name = f'{uuid.uuid4()}'
```